autopush/settings.py

```diff
@@ -206,15 +210,28 @@ def message(self, value):
     def create_initial_message_tables(self):
         """Initializes a dict of the initial rotating messages tables.
 
-        An entry for last months table, and an entry for this months table.
+        An entry for last months table, an entry for this months table,
+        and an entry for tomorrow, if it starts a new month.
 
         """
+        today = datetime.date.today()
         last_month = get_rotating_message_table(self._message_prefix, -1)
         this_month = get_rotating_message_table(self._message_prefix)
-        self.message_tables = {
-            last_month.table_name: Message(last_month, self.metrics),
-            this_month.table_name: Message(this_month, self.metrics),
-        }
+        self.current_month = today.month
+        self.current_msg_month = this_month.table_name
+        tomorrow = get_rotating_message_table(
+            self._message_prefix, 0, today + datetime.timedelta(days=1))
+        self.message_tables = defaultdict(MessageSink)
+        self.message_tables[last_month.table_name] = Message(last_month,
+                                                             self.metrics)
+        self.message_tables[this_month.table_name] = Message(this_month,
+                                                             self.metrics)
+        # Due to issues around mocking "datetime" and desire to
+        # reduce excessive hotpath function calls, we have to skip this
+        # line
+        if tomorrow.table_name != this_month.table_name:  # pragma: nocover
+            self.message_tables[tomorrow.table_name] = Message(
+                tomorrow, self.metrics)
```

Comment:
It should be in both, in cases where the app is started at the end of the month. thanks!